src/server/pfs/server/driver.go

```diff
@@ -150,7 +159,10 @@ func newLocalDriver(blockAddress string, etcdPrefix string) (*driver, error) {
 	return newDriver(blockAddress, []string{"localhost:32379"}, etcdPrefix, defaultCacheSize)
 }
 
-func (d *driver) getObjectClient() (*client.APIClient, error) {
+// TODO(msteffen): client initialization (both etcd and pachd) might be better
+// placed happen in server.go, near main(), so that we only pay the dial cost
+// once, and so that pps doesn't need to have its own initialization code
+func (d *driver) initializePachConn() error {
```

Comment:
I wonder if it makes sense to put this into `CheckIsAuthorized` to reduce boilerplate.